syntax = "proto3";
package com.fiberhome.fums.proto;
option java_package = "com.fiberhome.fums.proto";   
option java_outer_classname = "StbInspect";
import "CommonDefine.proto";

message StbInspectData
{	 
	string   messageType    = 1;
	string   stbIp          = 2;
	string   entityType     = 3;
	string   entityInstance = 4;
	EnumDefineData.AppTypeEnum appTypeCode = 5;
	string   operDirect     = 6;
	string   operName       = 7;
	string   operArgList    = 8;
	string   operRst        = 9;
	string   operOccurTime  = 10;
	string   desp           = 11;	
}

